ur_moveit_config/config/ur_servo.yaml

```diff
@@ -53,7 +56,7 @@ cartesian_command_in_topic: ~/delta_twist_cmds  # Topic for incoming Cartesian t
 joint_command_in_topic: ~/delta_joint_cmds # Topic for incoming joint angle commands
 joint_topic: /joint_states
 status_topic: ~/status # Publish status to this topic
-command_out_topic: /forward_velocity_controller/commands # Publish outgoing commands here
+command_out_topic: /joint_trajectory_controller/joint_trajectory # Publish outgoing commands here
```

Comment:
This is the default controller type that is launched from this package at this point in time. It works fine with Servo.